sentry-logback/src/main/java/io/sentry/logback/SentryAppender.java

```diff
@@ -137,6 +140,13 @@ protected void append(@NotNull ILoggingEvent eventObject) {
     return event;
   }
 
+  private String formatted(@NotNull ILoggingEvent loggingEvent) {
+    if (encoder != null) {
+      return new String(encoder.encode(loggingEvent), StandardCharsets.UTF_8);
```

Comment:
Would it make sense to wrap this with a try/catch just in case a faulty encoder has been configured so we still get the formatted message as fallback?